dtran/pipeline.py

```diff
@@ -177,3 +257,52 @@ def save(self, save_path: Union[str, Path]):
     @staticmethod
     def load(load_path: Union[str, Path]):
         pass
+
+
+def default_wrapper(cls: Type[IFunc], inputs: Set[str]) -> Type[IFunc]:
```

Comment:
how about inheritance vs wrapper function?

Reply:
I think it's good for now. We can discuss and formalize creation of Wrappers in the next update.